CHANGES.rst

```diff
@@ -107,6 +107,10 @@ astropy.utils
 astropy.visualization
 ^^^^^^^^^^^^^^^^^^^^^
 
+- Added a `log_a` argument to the `simple_norm` method. Similar to the
```

Comment:
you need to add double backticks here. Single ones are tried to parsed as links by sphinx and cause failures.

Reply:
Gotcha I will take care of it now